packages/elements-core/src/styles.css

```diff
@@ -90,6 +88,6 @@
   padding-right: 10px;
 }
 
-.sl-elements .HttpOperation__gutter {
+/* .sl-elements .HttpOperation__gutter {
```

Comment:
Not used anywhere. I suspect there is more dangling CSS in this file but haven't investigated

```suggestion
```